src/components/endpoint/endpoint_config.js

```diff
@@ -162,6 +163,10 @@ class EndpointConfigCtrl {
     return Object.keys(ids).length;
   }
 
+  totalChecks(monitor) {
+    return (30.5 * 24 * (3600/monitor.frequency) * this.collectorCount(monitor) / 1000000) + 0.5;
```

Comment:
Would be cool if we could show separate numbers for 28/29/30/31 day months
 Maybe not worth it but you know me I gotta be picky about things